brayns/engine/renderers/ProductionRenderer.h

```diff
@@ -36,4 +37,25 @@ class ProductionRenderer final : public Renderer
 protected:
     void commitRendererSpecificParams() final;
 };
+
+BRAYNS_JSON_ADAPTER_BEGIN(ProductionRenderer)
+BRAYNS_JSON_ADAPTER_GETSET(
+    "samples_per_pixel",
+    getSamplesPerPixel,
+    setSamplesPerPixel,
+    "Number of samples per pixel",
+    Required(false))
+BRAYNS_JSON_ADAPTER_GETSET(
+    "max_ray_bounces",
+    getMaxRayBounces,
+    setMaxRayBounces,
+    "Max ray bounces per sample",
+    Required(false))
+BRAYNS_JSON_ADAPTER_GETSET(
+    "background_color",
+    getBackgroundColor,
+    setBackgroundColor,
+    "Background color",
+    Required(false))
```

Comment:
Maybe a macro with theses properties as they are defined for all renderers (but with only two it is fine like that also).

Reply:
Yes, I duplicated because they were only 2. The macro can be defined to avoid duplication.